public/outreach.html

```diff
@@ -127,16 +130,16 @@ <h1 class="hyphenate border-bottom border-light" id="k12">K-12 Education</h1>
             </div>
             <div class="col-lg-6 bg-logo-green text-white px-rem3 py-rem5 d-flex flex-column justify-content-lg-around">
                <div class>
-                  <h1 class="hyphenate border-bottom border-light" id="knowledge_sharing">Knowledge sharing</h1>
+                  <h2 class="h1 hyphenate border-bottom border-light" id="knowledge_sharing">Knowledge sharing</h2>
                   <p>The Alaskan Arctic coast is the traditional home of the I&#xF1;upiaq, who make up the majority of the
                      population on the North Slope. In collaboration with the Bureau of Ocean Energy Management and
                      North Slope Borough Fish and Wildlife Department, we formed a panel of traditional knowledge holders
                      in Kaktovik, Alaska to advise us on emerging ideas and research plans and to act as a conduit for
                   knowledge exchange between BLE and the community.</p>
                   <p>We also share knowledge with local communities in-person through the Barrow Science Fair and
                      Kaktovik Community meetings, in print through our 
-                     <a class="text-white" rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank">newsletter The Lagoon Exchange</a>,
-                     and online via our <a class="text-white" rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank">Facebook page</a>.</p>
+                     <a class="text-white" rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank" aria-label="open in new tab">newsletter The Lagoon Exchange</a>,
+                     and online via our <a class="text-white" rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank" aria-label="open in new tab">Facebook page</a>.</p>
```

Comment:
Open BLE Facebook page in new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5